arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3817,7 +3817,7 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
     }
     std::vector<ServerID> dbServers = ci.getCurrentDBServers();
     std::vector<ServerID> lockedServers;
-    double lockWait(1);
+    double lockWait(14.0);
```

Comment:
There is potentially a reason for the timeout starting with a value of `14.0`, but it is totally unclear to me.
An explaining comment in the code would help.

Reply:
we fear that the shorter periodsjust do more harm than good. specifically on slow oasis nodes

Reply:
I have added an explaining comment and even adjusted the time one more time.